visualization/app/codeCharta/ui/ribbonBar/ribbonBar.e2e.ts

```diff
@@ -121,9 +122,29 @@ describe("RibbonBar", () => {
 		const areaPanel = "area-metric"
 
 		await ribbonBar.togglePanel(areaPanel)
-		expect(AreaSettingsPanelPageObject.isDynamicMarginEnabled()).toBeTruthy()
-		expect(await AreaSettingsPanelPageObject.toggleDynamicMargin()).toBeFalsy()
+		expect(AreaSettingsPanelPageObject.isDefaultMarginEnabled()).toBeTruthy()
+		expect(await AreaSettingsPanelPageObject.toggleDefaultMargin()).toBeFalsy()
 
 		expect(await ribbonBar.isPanelOpen(areaPanel)).toBeTruthy()
 	})
+
+	it("should check if reset button height matches the height of the last entry of color component", async () => {
+		const colorPanel = "color-metric"
+
+		await ribbonBar.togglePanel(colorPanel)
+		const boundingBoxCheckbox = await ColorSettingsPageObject.toggleInverColorBoundingBox()
+		const boundingBoxResetButton = await ColorSettingsPageObject.resetButtonBoundingBox()
+
+		expect(Math.round(boundingBoxCheckbox.y)).toEqual(Math.floor(boundingBoxResetButton.y))
```

Comment:
without trying it, I'm not sure about this equality check

Reply:
Yes, it seems to have a variance depending on the browser, I should increase the tolerance window, the idea is to check if the coordinates roughly match. +-5 px will not affect the component to much, but 150 like in the issue is a big problem.

Reply:
Thanks I will change it